fluentd/configs.d/user/fluent.conf

```diff
@@ -24,6 +24,13 @@
   @include configs.d/openshift/filter-syslog-record-transform.conf
   @include configs.d/openshift/filter-viaq-data-model.conf
   @include configs.d/openshift/filter-post-*.conf
+
+  # generate uuids at source mitigate creating duplicate records
+  # requires fluent-plugin-elasticsearch
+  <filter **>
+    @type elasticsearch_genid
+    hash_id_key _openshift_es_genid
+  </filter>
```

Comment:
Follow up to IRC discussion - this should go above filter-post-*.conf.   My successful (no duplicates) config was:

    <label @INGRESS>
    ## filters
      @include configs.d/openshift/filter-pre-*.conf
      @include configs.d/openshift/filter-retag-journal.conf
      @include configs.d/openshift/filter-k8s-meta.conf
      @include configs.d/openshift/filter-kibana-transform.conf
      @include configs.d/openshift/filter-k8s-flatten-hash.conf
      @include configs.d/openshift/filter-k8s-record-transform.conf
      @include configs.d/openshift/filter-syslog-record-transform.conf
      @include configs.d/openshift/filter-viaq-data-model.conf
      <filter **>
        @type elasticsearch_genid
        hash_id_key _openshift_es_genid
      </filter>
      @include configs.d/openshift/filter-post-*.conf
    </label>

Reply:
fixed